### PR TITLE
Use sbt-web 1.5.x despite staying on paradox 0.9.2

### PR DESCRIPTION
**Pull request by mkurz**
Same as in 
- https://github.com/apache/incubator-pekko-http/pull/486
- https://github.com/apache/incubator-pekko/pull/1135

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -76,7 +76,8 @@ lazy val pekkoPlugin = project
     addSbtPlugin(
       // When updating the sbt-paradox version,
       // remember to also update project/plugins.sbt
-      "com.lightbend.paradox" % "sbt-paradox" % "0.10.3"),
+      ("com.lightbend.paradox" % "sbt-paradox" % "0.10.3").exclude("com.typesafe.sbt", "sbt-web")),
```
Comment (mkurz) [inline build.sbt:79]: Why `0.10.3` here? Shouldn't that be either downgraded to 0.9.2 or upgraded to 0.10.6?

Comment (mdedetrich) [reply build.sbt:79]: This is a mistake, can you fix it?

Comment (mkurz) [reply build.sbt]: Pushed

**Review by mdedetrich — APPROVED** [review]
Thanks, lgtm

**Pull request merged**